at_secondary/at_secondary_server/test/notify_verb_test.dart

```diff
@@ -787,7 +783,7 @@ Future<void> tearDownFunc() async {
   AtNotificationMap.getInstance().clear();
   await AtNotificationKeystore.getInstance().close();
   if (isExists) {
-    Directory('test/hive').deleteSync(recursive: true);
+    await Directory('test/hive').delete(recursive: true);
```

Comment:
deleteSync and await..delete ..both are same

Reply:
Yeah actually that doesn't matter. But yeah that's fine for anything.